eng/pipelines/generate-sdk-for-java.yml

```diff
@@ -0,0 +1,58 @@
+parameters:
```

Comment:
What is the name of the pipeline going to be on the DevOps side? We should try and name this file to match that.

Reply:
I'm fine with having one yml per language I just want to know what we plan to name the pipelines.

Reply:
Even with one yml per language, user can still override the specRepoUrl and sdkRepoUrl while queuing a run. This could lead to confusion where a specific language pipeline ends up generating the SDK for another different language.

Reply:
If we do want yml file per language we should remove the sdkRepoUrl parameter as that will be hard-coded.